### PR TITLE
monthly_commentのモデル作成

### PR DESCRIPTION
**Pull request by Nobuhiko-Kudo**

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -70,6 +70,16 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "monthly_comments", force: :cascade do |t|
```
Comment (bakunyo) [inline db/schema.rb]: **[rails_best_practices]**
- always add db index (monthly_comments => [monthly_report_id]) [:ref](http://rails-bestpractices.com/posts/2010/07/24/always-add-db-index/)

File: db/migrate/20150924122138_create_monthly_comments.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class CreateMonthlyComments < ActiveRecord::Migration
```
Comment (bakunyo) [inline db/migrate/20150924122138_create_monthly_comments.rb]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: db/migrate/20150924122138_create_monthly_comments.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+  def change
+    create_table :monthly_comments do |t|
```
Comment (bakunyo) [inline db/migrate/20150924122138_create_monthly_comments.rb]: **[reek]**
- [UncommunicativeVariableName] has the variable name 't'

File: db/migrate/20150924122138_create_monthly_comments.rb [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+      t.belongs_to :user, index: true
```
Comment (bakunyo) [inline db/migrate/20150924122138_create_monthly_comments.rb]: **[reek]**
- [FeatureEnvy] refers to t more than self

File: db/migrate/20150924122138_create_monthly_comments.rb [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+      t.text :comment
```
Comment (bakunyo) [inline db/migrate/20150924122138_create_monthly_comments.rb]: **[reek]**
- [FeatureEnvy] refers to t more than self

File: db/migrate/20150924122138_create_monthly_comments.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+      t.integer :monthly_report_id, null: false
```
Comment (bakunyo) [inline db/migrate/20150924122138_create_monthly_comments.rb]: **[reek]**
- [FeatureEnvy] refers to t more than self

File: db/migrate/20150924122138_create_monthly_comments.rb [hunk 7]
```diff
@@ -0,0 +1,11 @@
...
+
+      t.timestamps null: false
```
Comment (bakunyo) [inline db/migrate/20150924122138_create_monthly_comments.rb]: **[reek]**
- [FeatureEnvy] refers to t more than self

**Comment by bakunyo** [comment]
CIテスト落ちちゃってるので、最新から切りなおしてもらえると嬉しいです〜:D

File: db/schema.rb [hunk 8]
```diff
@@ -56,6 +56,14 @@
...
```
Comment (freezer530000) [inline db/schema.rb]: **[rails_best_practices]**
- always add db index (monthly_comments => [user_id]) [:ref](http://rails-bestpractices.com/posts/2010/07/24/always-add-db-index/)
- always add db index (monthly_comments => [monthly_report_id]) [:ref](http://rails-bestpractices.com/posts/2010/07/24/always-add-db-index/)

File: db/migrate/20150927113409_create_monthly_comments.rb [hunk 9]
```diff
@@ -0,0 +1,11 @@
+class CreateMonthlyComments < ActiveRecord::Migration
```
Comment (freezer530000) [inline db/migrate/20150927113409_create_monthly_comments.rb]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: db/migrate/20150927113409_create_monthly_comments.rb [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+  def change
+    create_table :monthly_comments do |t|
```
Comment (freezer530000) [inline db/migrate/20150927113409_create_monthly_comments.rb]: **[reek]**
- [UncommunicativeVariableName] has the variable name 't'

File: db/migrate/20150927113409_create_monthly_comments.rb [hunk 11]
```diff
@@ -0,0 +1,11 @@
...
+      t.belongs_to :user, null: false
```
Comment (freezer530000) [inline db/migrate/20150927113409_create_monthly_comments.rb]: **[reek]**
- [FeatureEnvy] refers to t more than self

File: db/migrate/20150927113409_create_monthly_comments.rb [hunk 12]
```diff
@@ -0,0 +1,11 @@
...
+      t.text :comment
```
Comment (freezer530000) [inline db/migrate/20150927113409_create_monthly_comments.rb]: **[reek]**
- [FeatureEnvy] refers to t more than self

File: db/migrate/20150927113409_create_monthly_comments.rb [hunk 13]
```diff
@@ -0,0 +1,11 @@
...
+      t.belongs_to :monthly_report, null: false
```
Comment (freezer530000) [inline db/migrate/20150927113409_create_monthly_comments.rb]: **[reek]**
- [FeatureEnvy] refers to t more than self

File: db/migrate/20150927113409_create_monthly_comments.rb [hunk 14]
```diff
@@ -0,0 +1,11 @@
...
+
+      t.timestamps null: false
```
Comment (freezer530000) [inline db/migrate/20150927113409_create_monthly_comments.rb]: **[reek]**
- [FeatureEnvy] refers to t more than self

File: app/models/monthly_comment.rb [hunk 15]
```diff
@@ -0,0 +1,14 @@
...
+#
+#  id                :integer          not null, primary key
+#  user_id           :integer          not null
+#  comment           :text
+#  monthly_report_id :integer          not null
+#  created_at        :datetime         not null
+#  updated_at        :datetime         not null
+#
+
+class MonthlyComment < ActiveRecord::Base
```
Comment (bakunyo) [inline app/models/monthly_comment.rb]: あ…すみません先に気づけばよかったですが、`MonthlyReportTag`に合わせて`MonthlyReportComment`にして頂けるとしっくりきます！

File: db/migrate/20150928132655_create_monthly_report_comments.rb [hunk 16]
```diff
@@ -0,0 +1,11 @@
+class CreateMonthlyReportComments < ActiveRecord::Migration
```
Comment (freezer530000) [inline db/migrate/20150928132655_create_monthly_report_comments.rb:1]: **[reek]**
- [IrresponsibleModule] has no descriptive comment

File: db/migrate/20150928132655_create_monthly_report_comments.rb [hunk 17]
```diff
@@ -0,0 +1,11 @@
...
+  def change
+    create_table :monthly_report_comments do |t|
```
Comment (freezer530000) [inline db/migrate/20150928132655_create_monthly_report_comments.rb:3]: **[reek]**
- [UncommunicativeVariableName] has the variable name 't'

File: db/schema.rb [hunk 18]
```diff
@@ -56,6 +56,14 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "monthly_report_comments", force: :cascade do |t|
```
Comment (freezer530000) [inline db/schema.rb:59]: **[rails_best_practices]**
- always add db index (monthly_report_comments => [user_id]) [:ref](http://rails-bestpractices.com/posts/2010/07/24/always-add-db-index/)
- always add db index (monthly_report_comments => [monthly_report_id]) [:ref](http://rails-bestpractices.com/posts/2010/07/24/always-add-db-index/)

File: db/migrate/20150928132655_create_monthly_report_comments.rb [hunk 19]
```diff
@@ -0,0 +1,11 @@
...
+      t.belongs_to :user, null: false
```
Comment (freezer530000) [inline db/migrate/20150928132655_create_monthly_report_comments.rb:4]: **[reek]**
- [FeatureEnvy] refers to t more than self (maybe move it to another class?)

File: db/migrate/20150928132655_create_monthly_report_comments.rb [hunk 20]
```diff
@@ -0,0 +1,11 @@
...
+      t.text :comment
```
Comment (freezer530000) [inline db/migrate/20150928132655_create_monthly_report_comments.rb:5]: **[reek]**
- [FeatureEnvy] refers to t more than self (maybe move it to another class?)

File: db/migrate/20150928132655_create_monthly_report_comments.rb [hunk 21]
```diff
@@ -0,0 +1,11 @@
...
+      t.belongs_to :monthly_report, null: false
```
Comment (freezer530000) [inline db/migrate/20150928132655_create_monthly_report_comments.rb:6]: **[reek]**
- [FeatureEnvy] refers to t more than self (maybe move it to another class?)

File: db/migrate/20150928132655_create_monthly_report_comments.rb [hunk 22]
```diff
@@ -0,0 +1,11 @@
...
+
+      t.timestamps null: false
```
Comment (freezer530000) [inline db/migrate/20150928132655_create_monthly_report_comments.rb:8]: **[reek]**
- [FeatureEnvy] refers to t more than self (maybe move it to another class?)

**Comment by bakunyo** [comment]
LGTM
SideCIのは別Issueに切り出したのでそちらで対応しましょう〜

**Comment by tamata78** [comment]
月報コメントはユーザーと月報に紐づけられているので問題ないと思います。

**Pull request merged**